### ISSUE TITLE
Doesn't support the custom properties when checking the value of an attribute derived from CSS on SVG.

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#364]
```html
<svg>
  <rect fill="var(--custom-property)" />
</svg>
```

The `invalid-attr` rule reports the violation in `var(`.
Because CSSTree doesn't support the custom properties yet: https://github.com/csstree/csstree/issues/62

**Comment by YusukeHirao** [comment]
Resolved: #2073
